yt/data_objects/tests/test_profiles.py

```diff
@@ -293,3 +293,13 @@ def DM_in_cell_mass(field, data):
                                 weight_field=("gas", "DM_cell_mass"))
 
     assert not np.any(np.isnan(profile['gas', 'radial_velocity']))
+
+def test_profile_override_limits():
+    ds = fake_random_ds(64, nprocs = 8, fields = _fields, units = _units)
+
+    sp = ds.sphere(ds.domain_center, (10, 'kpc'))
+    obins = np.linspace(-5,5,10)
+    profile = yt.create_profile(sp,
+                                [ "density"],["temperature"],
+                                override_bins={"density":(obins, "g/cm**3")})
```

Comment:
you're only testing `Profile1D` here. It may be worthwhile to explicitly test `Profile2D` and `Profile3D` as well, since there's code to support this in all three.

Reply:
I agree with @ngoldbaum here